package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "prestart": "npm install",
     "start": "npm run dev",
-    "dev": "ng serve --host='0.0.0.0' -o",
+    "dev": "ng serve --host='0.0.0.0' --port=4301 -o",
```

Comment:
Remove this line.   This isn't high priority and doesn't need to be done before the build today, but it should be removed.

1. we handle ports in angular.json file
2. Each project should have their own unique port. I think MSPs is already 4300. Not sure if 4301 is taken (I know 4201 is prime, 4209 is sitereg)